package.json

```diff
@@ -109,10 +107,8 @@
     "gh-pages": "2.0.1",
     "gunzip-maybe": "1.4.1",
     "hashish": "0.0.4",
-    "isparta": "4.1.1",
     "jest": "24.7.1",
     "jest-raw-loader": "1.0.1",
-    "json-loader": "0.5.7",
```

Comment:
imho this is being used for imports like `import badwords from './badwords.json';` but hmm… looking a the package and reading "⚠️ Since webpack >= v2.0.0, importing of JSON files will work by default. You might still want to use this if you use a custom file extension. See the v1.0.0 -> v2.0.0 Migration Guide for more information" - I guess this is safe to remove :laughing: